sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/namedExpressions.scala

```diff
@@ -324,31 +324,28 @@ case class AttributeReference(
  * A place holder used when printing expressions without debugging information such as the
  * expression id or the unresolved indicator.
  */
-case class PrettyAttribute(
+case class PrettyNamedExpression(
     name: String,
     dataType: DataType = NullType)
-  extends Attribute with Unevaluable {
```

Comment:
can we still call it `PrettyAttribute` and extends `Attribute`?

Reply:
You mean the latest commit? I just renamed this because `Alias` is not `Attribute`.